cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -10410,3 +10410,99 @@ struct_init.cpp:
+#   13|     r13_2(bool) = CompareEQ         : r13_8, r13_1
+#   13|     v13_3(void) = ConditionalBranch : r13_2
+#-----|   False -> Block 2
+#-----|   True -> Block 3
+
+#   13|   Block 2
+#   13|     r13_4(int)  = Constant[1]       : 
+#   13|     r13_5(bool) = CompareEQ         : r13_8, r13_4
+#   13|     v13_6(void) = ConditionalBranch : r13_5
+#-----|   True -> Block 5
+
+#   13|   Block 3
+#   13|     v13_7(void) = Unwind : 
+#-----|   Goto -> Block 6
+
+#   13|   Block 4
+#   13|     r13_8(int)   = Constant[0]       : 
+#   13|     r13_9(int)   = Constant[-1]      : 
+#   13|     r13_10(bool) = CompareEQ         : r13_8, r13_9
+#   13|     v13_11(void) = ConditionalBranch : r13_10
+#-----|   False -> Block 1
+#-----|   True -> Block 3
+
+#   14|   Block 5
+#   14|     r14_1(glval<unknown>) = FunctionAddress[sink] : 
+#   14|     r14_2(glval<int>)     = VariableAddress[x]    : 
+#   14|     r14_3(int)            = Load[x]               : &:r14_2, ~m?
+#   14|     v14_4(void)           = Call[sink]            : func:r14_1, 0:r14_3
+#   14|     mu14_5(unknown)       = ^CallSideEffect       : ~m?
+#-----|   Goto -> Block 6
+
+#   16|   Block 6
+#   16|     v16_1(void) = NoOp         : 
+#    6|     v6_4(void)  = ReturnVoid   : 
+#    6|     v6_5(void)  = AliasedUse   : ~m?
+#    6|     v6_6(void)  = ExitFunction : 
+
+#   18| void g()
+#   18|   Block 0
+#   18|     v18_1(void)           = EnterFunction                  : 
+#   18|     mu18_2(unknown)       = AliasedDefinition              : 
+#   18|     mu18_3(unknown)       = InitializeNonLocal             : 
+#   19|     r19_1(glval<int>)     = VariableAddress[x]             : 
+#   19|     mu19_2(int)           = Uninitialized[x]               : &:r19_1
+#   19|     r19_3(glval<int>)     = VariableAddress[y]             : 
+#   19|     r19_4(int)            = Constant[0]                    : 
+#   19|     mu19_5(int)           = Store[y]                       : &:r19_3, r19_4
+#   21|     r21_1(glval<unknown>) = FunctionAddress[ProbeFunction] : 
+#   21|     r21_2(int)            = Constant[0]                    : 
+#   21|     v21_3(void)           = Call[ProbeFunction]            : func:r21_1, 0:r21_2
+#   21|     mu21_4(unknown)       = ^CallSideEffect                : ~m?
+#   22|     r22_1(glval<int>)     = VariableAddress[y]             : 
+#   22|     r22_2(int)            = Load[y]                        : &:r22_1, ~m?
+#   22|     r22_3(glval<int>)     = VariableAddress[x]             : 
+#   22|     mu22_4(int)           = Store[x]                       : &:r22_3, r22_2
+#   23|     r23_1(glval<unknown>) = FunctionAddress[ProbeFunction] : 
+#   23|     r23_2(int)            = Constant[0]                    : 
+#   23|     v23_3(void)           = Call[ProbeFunction]            : func:r23_1, 0:r23_2
+#   23|     mu23_4(unknown)       = ^CallSideEffect                : ~m?
+#   26|     r26_1(glval<unknown>) = FunctionAddress[sink]          : 
```

Comment:
I guess technically a new block should start here, or we need a special instruction here, as this point is directly reachable from all the statements in the `__try` block. Again this is probably now very relevant for dataflow at this point, so I propose to file an issue for this.

Reply:
Yeah, I completely agree. There's probably a large room for design here. I'll create an issue for it.